VoltAir/qml/ui/LoadingScreen.qml

```diff
@@ -108,6 +108,6 @@ UiComponent {
         width: loading.width
         height: 0.25 * loading.height
 
-        textElement.text: "Loading"
+        textElement.text: TR.value("Loading")
```

Comment:
Hi Jo, 
yes, TR is a class that handles the translations. Internally has a dictionary, which, when invoked the value function, translates the enclosed text.
Currently, if not translated a TBT is preppended. (TBT: To be translated).